cli/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hdp-cli"
-version = "0.2.0"
+version = "0.2.5"
```

Comment:
u can specify same by workspace.version and workspace.edition

Reply:
done!